src/index.ts

```diff
@@ -1294,25 +1294,34 @@ export class VerificationError extends Error {}
  * check
  * @param key                   A [[Buffer]] containing the key to check
  * @param witness               The [[Witness]] to verify
+ * @param exist                 For existence proofs exist has to be true
```

Comment:
Is this necessary? The Witness already has the value, so you just need to check Witness.value === null